pkg/systemd/host.js

```diff
@@ -108,16 +110,23 @@ function ServerTime() {
      * behavior and formatting of a Date() object in the absence of
      * IntlDateFormat and  friends.
      */
-    Object.defineProperty(self, 'now', {
+    Object.defineProperty(self, 'utc_fake_now', {
```

Comment:
Yes it's an improvement in the right direction. Previously it was hidden complexity. Now it's called out.